tests/Mono.Android-Tests/Xamarin.Android.Net/AndroidMessageHandlerTests.cs

```diff
@@ -248,6 +248,22 @@ public async Task AndroidMessageHandlerFollows308PermanentRedirect ()
 			Assert.AreEqual ("https://www.microsoft.com/", result.RequestMessage.RequestUri.ToString ());
 		}
 
+		[Test]
+		public async Task AndroidMessageHandlerSendsClientCertificate ()
```

Comment:
a'la [my previous comment](https://github.com/xamarin/xamarin-android/pull/8961/files#r1610537993), we should update this test to include setting `ClientCertificateOptions`:

```csharp
[Test]
public async Task AndroidMessageHandlerSendsClientCertificate([Values(true, false)] bool manualOptions)
{
    string testClientCertificate = …;
    using var certificate = new X509Certificate(Convert.FromBase64String (testClientCertificate), "pass");
    using var handler = new AndroidMessageHandler {
        ClientCertificates = { certificate },
        ClientCertificateOptions = manualOptions ? ClientCertificateOptions.Manual : ClientCertificateOptions.Automatic,
    };
    using var client = new HttpClient (handler);

    try {
        var response = await client.GetAsync ("https://corefx-net-tls.azurewebsites.net/EchoClientCertificate.ashx");
        var content = await response.EnsureSuccessStatusCode ().Content.ReadAsStringAsync ();
        if (!manualOptions) AssertNotReached();
    }
    catch (Exception e) {
        if (manualOptions) AssertNotReached();

        // assert that e is whatever it should be (InvalidOperationException?)
    }
}